website/docs/r/group.html.markdown

```diff
@@ -44,6 +44,28 @@ In addition to the attributes listed above, the following attribute is also expo
 
 ## Additional Examples
 
+### Updating User Group Membership Management in Terraform
+
+### Overview
+There is a potential race condition within Terraform when managing user accounts and their respective group memberships. A user might be deleted before Terraform disassociates them from a user group. This can lead to an error during terraform apply because the user ID no longer exists when the group resource is being updated.
```

Comment:
```suggestion
There is a potential race condition within Terraform when managing user accounts and their respective group memberships. A user might be deleted before Terraform disassociates them from a user group. This can lead to an error during `terraform apply` because the user ID no longer exists when the group resource is being updated.
```